src/python/pants/backend/docgen/tasks/generate_pants_reference.py

```diff
@@ -54,7 +54,7 @@ def execute(self):
     def _gen_reference(self):
         def get_scope_data(scope):
             ret = []
-            for si in ScopeInfoIterator(self.context.options.known_scope_to_info).iterate([scope]):
+            for si in ScopeInfoIterator(self.context.options.known_scope_to_info).iterate({scope}):
```

Comment:
See below comments on `Iterable[str]` vs. `Set[str]`. I don't think we get much benefit from this change.